### PR TITLE
Filter for bl:Entity class

### LINKED ISSUE #51: update missing-biolink-term check to exclude 'entity'
There is a new term 'entity' above 'named thing'. We should filter this out from the check for missing mappings.

### PR DESCRIPTION
**Pull request by Shalsh23**
Fixes #51

### CODE REVIEW

**Comment by rajshruti18** [comment]
There is no exception to the relation that 'NamedThing' is a subclass of 'Entity' as we got the same results even if we removed either of the two from the query.

**Comment by Shalsh23** [comment]
For reference: [https://api.triplydb.com/s/tWfhlT4Zs](url)

**Comment by balhoff** [comment]
> There is no exception to the relation that 'NamedThing' is a subclass of 'Entity' as we got the same results even if we removed either of the two from the query.

@rajshruti18 I think that makes sense, since I fixed the recent issues I came across via #50.

**Review by balhoff — CHANGES_REQUESTED** [review]
I would like this to be a little more clever. Instead of hardcoding a list, make it enforce that some ?bl_type exists that is more specific than NamedThing in the is_a hierarchy.

File: sparql/reports/owl-missing-biolink-term.rq [hunk 1]
```diff
@@ -12,7 +12,8 @@ WHERE {
   FILTER NOT EXISTS {
 	?n0 rdf:type ?bl_type .
     ?bl_type rdf:type metatype:ClassDefinition .
-    FILTER(?bl_type != bl:NamedThing)
+    OPTIONAL{ bl:NamedThing rdfs:subclassOf ?entity } .
```
Comment (Shalsh23) [inline sparql/reports/owl-missing-biolink-term.rq:15]: Should I add a constraint that `?entity rdf:type blml:types/ClassDefinition`? (Ref: [)](https://api.triplydb.com/s/TAkcLrX_c)

**Comment by balhoff** [comment]
Constraints to keep in the NOT EXISTS:

```sparql
FILTER(?bl_type != bl:NamedThing)
?bl_type rdfs:subClassOf bl:NamedThing
```

**Review by balhoff — APPROVED** [review]

**Pull request merged**